src/main/java/lcduke/TaskList.java

```diff
@@ -0,0 +1,182 @@
+                        part1 = "03";
+                        break;
+                    case("Feb"):
+                        part1 = "02";
+                        break;
+                    default:
+                        part1 = "01";
+                }
+                if (part2.length() < 2){
+                    part2 = "0" + part2;
+                }
+                temp = part3 + "-" + part1 + "-" + part2;
+                this.deadline("deadline" + totalTasks[i].substring(totalTasks[i].indexOf(" "), totalTasks[i].indexOf("by:") - 1)
+                        + "/by " + temp);
+
+            } else {
+                String temp = totalTasks[i].substring(totalTasks[i].indexOf("at:") + 4, totalTasks[i].length() - 1);
+                String part1 = temp.substring(0, 4);
+                String part2 = temp.substring(4);
+                part2 = part2.substring(0, part2.indexOf(" "));
+                String part3 = temp.substring(temp.length()-4, temp.length());
+
+                switch(part1){
+                    case("Dec"):
+                        part1 = "12";
+                        break;
+                    case("Nov"):
+                        part1 = "11";
+                        break;
+                    case("Oct"):
+                        part1 = "10";
+                        break;
+                    case("Sep"):
+                        part1 = "09";
+                        break;
+                    case("Aug"):
+                        part1 = "08";
+                        break;
+                    case("Jul"):
+                        part1 = "07";
+                        break;
+                    case("Jun"):
+                        part1 = "06";
+                        break;
+                    case("May"):
+                        part1 = "05";
+                        break;
+                    case("Apr"):
+                        part1 = "04";
+                        break;
+                    case("Mar"):
+                        part1 = "03";
+                        break;
+                    case("Feb"):
+                        part1 = "02";
+                        break;
+                    default:
+                        part1 = "01";
+                }
+                if (part2.length() < 2){
+                    part2 = "0" + part2;
+                }
+                temp = part3 + "-" + part1 + "-" + part2;
+                System.out.println(temp);
+                this.event("event" + totalTasks[i].substring(totalTasks[i].indexOf(" "), totalTasks[i].indexOf("at:") - 1)
+                        + "/at " + temp);
+            }
+            if(totalTasks[i].contains("\u2713")){
+                this.totalTasks[i].markAsDone();
+            }
+        }
+    }
+
+    public void delete(String userInput){
+        String deleteSelect = userInput.substring(7);
+        int deleteSelectNo = parseInt(deleteSelect) - 1;
+        if (deleteSelectNo != totalTasksCount) {
```

Comment:
You should also handle the situation where user tries to delete invalid task number